README.md

```diff
@@ -158,6 +158,10 @@ The first argument for `route()` is a Lua table with one or more keys:
 * `path` - route path, as described earlier.
 * `name` - route name.
 * `method` - method on the route like `POST`, `GET`, `PUT`, `DELETE`
+* `log_level` - type of logger specified for logging request info.
+  Any digit from range 1 - 6 is allowed, where: `1` is disable logging - `6` is `log.verbose`.
+  By default, `log_level` is equal to 5 (`log.info`). If `log_requests` option is `false` this parameter will be ignored. 
```

Comment:
> If `log_requests` option is `false` this parameter will be ignored. 
This is not true. According to your code, `log_level` just rewrites a log function and doesn't depend on `log_requests` value anyhow.